Lib/test/test_builtin.py

```diff
@@ -772,6 +772,48 @@ def test_exec_redirected(self):
         finally:
             sys.stdout = savestdout
 
+    def test_exec_closure(self):
+        result = 0
+        def make_closure_functions():
+            a = 2
+            b = 3
+            c = 5
+            def two_freevars():
```

Comment:
Done:

- Renamed `three_freevars` to `four_freevars`.
- Then, renamed `two_freevars` to `three_freevars`.
- Added tests passing in a correctly-sized list of cell vars instead of a tuple, and a tuple of the right length with one non-cell-var entry.